MQ2PluginManager.cpp

```diff
@@ -4,30 +4,37 @@
 	around until things worked. There may be some residual code that is not even required by the
 	plugin in here, but if it is, I'm unaware of it. Chances are some of these things I just deleted
 	entirely and tried to run after a succesful compile only to crash and therefore was readded.
+
+	July  9, 2024 Grimmier  -- Added ImGui Window to the mqsettings Panel.
+	July 30, 2024 Grimmier  -- Removed the in game XML window and replaced with a IMGUI window.
+							-- Added a toggle button to the mqsettings panel window.
+							-- Switched the /pluginman command to toggle the IMGUI window instead of the old XML.
 */
 
 #include <mq/Plugin.h>
 #include "CPluginTree.h"
-#include "CPluginToolWnd.h"
+#include "imgui/ImGuiUtils.h"
+#include <imgui_internal.h>
```

Comment:
Why is the internal header being included here?

Reply:
oops i was using easyfind plugin as a check to make sure i was creating the standalone window correctly.